<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));
color: #FFF333;
font: 16pt &quot;Rage Italic&quot;;</string>
  </property>
  <widget class="QLineEdit" name="lineEdit_4">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>70</y>
     <width>321</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255, 0);
color: rgb(0, 0, 127);
text-align: center;
border:  rgb(255, 255, 255, 0);
font-size: 40px</string>
   </property>
   <property name="text">
    <string>          Welcome</string>
   </property>
  </widget>
  <widget class="QPushButton" name="login">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>390</y>
     <width>191</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 20px;
color: rgb(0, 0, 127)</string>
   </property>
   <property name="text">
    <string>Log in</string>
   </property>
  </widget>
  <widget class="QPushButton" name="create_new_account">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>470</y>
     <width>191</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 20px;
color: rgb(0, 0, 127)</string>
   </property>
   <property name="text">
    <string>Create new account</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_6">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>170</y>
     <width>411</width>
     <height>131</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255, 0);
color: rgb(0, 0, 127);
text-align: center;
border:  rgb(255, 255, 255, 0);
font-size: 30px</string>
   </property>
   <property name="text">
    <string>    Please sign in or create new account</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
